akka-remote/src/main/scala/akka/remote/serialization/ArteryControlMessageSerializer.scala

```diff
@@ -144,17 +146,25 @@ private[akka] class ArteryControlMessageSerializer(val system: ExtendedActorSyst
   def serializeSystemMessageEnvelope(env: SystemMessageDelivery.SystemMessageEnvelope): ArteryControlFormats.SystemMessageEnvelope = {
     val msg = MessageSerializer.serialize(system, env.message)
```

Comment:
I already changed it: the code doesn't use the actual bytes of the `SerializedMessage` any more but sticks them into our own fields below. If you think it is a strange pattern to use a protobuf class as a "struct" without actually using the (de)serialization methods, I'll think about another solution.

That's also what I meant in an above comment about "sharing the infrastructure". It seems `MiscMessageSerializer` already copied code from `MessageSerializer` (or the other way round) which I wanted to avoid.